src/morse/polling.clj

```diff
@@ -21,22 +21,34 @@
    Returns channel with updates from Telegram"
   [running token opts]
   (let [updates (a/chan)
+        updates-caller (or (:get-updates-fn opts) api/get-updates)
```

Comment:
if this is only for testing, then I recommend to do `with-redefs` on `get-updates` in tests

Reply:
For some reason `with-redefs` didn't work out for me from the first place in this scenario (maybe go block is the reason) - but it would definitely look nicer here.